src/Components/Reports/Util.js

```diff
@@ -0,0 +1,36 @@
+import { get } from 'lodash';
+
+export const responseToPDFData = (data) => {
+    const systemsRowsData = [];
+    const rowKeys = ['display_name', 'os', 'performance_utilization.cpu', 'performance_utilization.memory', 'performance_utilization.max_io',
+        'number_of_suggestions', 'state', 'reported_date'];
+
+    data.map((systemItem) => {
+        let rowValueArr = [];
+        rowKeys.map((rowKey) =>{
+            let rowValue =  rowKey === 'reported_date' ? '03 Mar 2022 06:58 UTC'  : get(systemItem, rowKey, '').toString();
```

Comment:
Api endpoint response will send report_date in `DD-MM-YYYY` format (not with time currently). Is that okay? Because in the report pdf I see last seen in this format -> `DD Mar YYYY HH:MM UTC`.

Reply:
```suggestion
            let rowValue =  rowKey === 'report_date' ? '03 Mar 2022 06:58 UTC'  : get(systemItem, rowKey, '').toString();
```
I would suggest to consider time as well. Btw, backend sends key `report_date`.
For reference -  https://console.redhat.com/docs/api/ros#operations-default-getSystemDetails

@PreetiW - As currently backend sends just date value, will frontend handle such values without an error?